src/client/components/App/index.js

```diff
@@ -46,9 +49,15 @@ const App = () => {
       .then(res => {
         const { course: c, roles: r, userid: u, onCampus: oc } = res.data;
         setCourse(c);
-        setRoles(r);
         setUserid(u);
         setOnCampusStatus(oc);
+        if (r && (r.includes('admin') || r.includes('administrator'))) {
+          setIsAdmin(true);
+          setIsInstructorOrAdmin(true);
+        }
+        if (r && r.includes('instructor')) {
```

Comment:
Do we also need `r.includes('teacher')`? I see that the previous code checked for 'teacher' too

Reply:
I've only seen instructor. Several weeks ago I checked for instructor and teacher because I didn't know what roles are possible..

Reply:
@maxpluez @chern The roles possible are listed here: https://www.imsglobal.org/spec/lti/v1p3/#role-vocabularies

I don't see a Teacher role.